src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -33,6 +33,14 @@ private void Awake()
     {
         _sequences = new List<BotSequence>();
 
+        // the path to sequences varies depending on if the overaly is used within the Unity editor
+        // or within an actual build of the game
+#if UNITY_EDITOR
+        _sequencePath = "Assets/RegressionGames/Resources/BotSequences";
+#else
+        _sequencePath = Application.persistentDataPath + "/BotSequences";
+#endif
```

Comment:
@RG-nAmKcAz  does this fix the issue you pointed out?
`3. RGSequenceManager assumes that we have file system access to a directory of sequences.  But again here, in production runtime these will be resources in the game, NOT a directory`

Reply:
Not fully

You'll have to handle resources for production.

You can steal/borrow some code from `BotSequence.LoadJsonResourcesFromDirectory` to handle loading every json from a path (whether as a resource or persistent file).  However, you can't use that method directly as it will only load from the filesystem if any file is found there, as opposed to providing the combined set of filesystem overrides + resources built into the game

Reply:
I thiiiink I have it figured out based on that function you mentioned. Its split into 3 cases:
- In the Unity Editor, use the hardcoded path: `Assets/RegressionGames/Resources/BotSequences`
- In a build, check to see if the persistent data path exists: `Application.persistentDataPath` + `/` + `BotSequences` exists
- In a build with nothing in the persistent data path? Use the `BotSequences` directory